hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/txns/crypto/CryptoUpdateTransitionLogic.java

```diff
@@ -160,6 +160,9 @@ private ResponseCodeEnum sanityCheck(AccountID target, HederaAccountCustomizer c
             if (newMax < ledger.alreadyUsedAutomaticAssociations(target)) {
                 return EXISTING_AUTOMATIC_ASSOCIATIONS_EXCEED_GIVEN_LIMIT;
             }
+            if (newMax > dynamicProperties.maxAutoAssociations()) {
+                return REQUESTED_NUM_AUTOMATIC_ASSOCIATIONS_EXCEEDS_ASSOCIATION_LIMIT;
+            }
             if (dynamicProperties.areTokenAssociationsLimited() && newMax > dynamicProperties.maxTokensPerAccount()) {
                 return REQUESTED_NUM_AUTOMATIC_ASSOCIATIONS_EXCEEDS_ASSOCIATION_LIMIT;
             }
```

Comment:
2nd time I've seen these six lines. Couldn't it go into `CryptoCreateChecks` like `tooManyAutoAssociations` (in the next file)?

Reply:
Hmm but this is `CryptoUpdate`...not sure how to do this refactor easily.

In modularized codebase, I am making sure such things don't get duplicated. 🙏